### PR TITLE
chore: Remove crud.less from Datasource

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
Removes the crud.less file from the Datasource component

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Go to Explore
2. Open the Edit dataset modal
3. Make sure there are no visual changes

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/Datasource/CollectionTable.tsx [hunk 1]
```diff
@@ -102,9 +101,28 @@ const CrudTableWrapper = styled.div<{ stickyHeader?: boolean }>`
...
-  }
+  ${({ theme }) => `
+    th span {
+      vertical-align: ${theme.gridUnit * -2}px;
+    }
+    .text-right {
+      text-align: right;
+    }
+    .empty-collection {
+      padding: ${theme.gridUnit * 2 + 2}px;
```
Comment (michael-s-molina) [inline superset-frontend/src/components/Datasource/CollectionTable.tsx:112]: I noticed that you removed the `.control-label` and `.frame` classes. It was intended?

Comment (geido) [reply superset-frontend/src/components/Datasource/CollectionTable.tsx:112]: @michael-s-molina yes I checked and those two are unused

Comment (michael-s-molina) [reply superset-frontend/src/components/Datasource/CollectionTable.tsx:112]: No. I just wanted to confirm that it was removed intentionally.

**Review by michael-s-molina — APPROVED** [review]
LGTM

**Pull request merged**